src/Evolution/DiscontinuousGalerkin/SlopeLimiters/HwenoModifiedSolution.hpp

```diff
@@ -164,28 +164,31 @@ secondary_neighbors_to_exclude_from_fit(
         get<::Tags::Mean<Tag>>(neighbor_and_data.second.means)[tensor_index] -
         local_mean);
   };
-  const auto max_difference_neighbor_and_data = *alg::max_element(
-      neighbor_data,
-      [&mean_difference](
-          const std::pair<std::pair<Direction<VolumeDim>, ElementId<VolumeDim>>,
-                          Package>& lhs,
-          const std::pair<std::pair<Direction<VolumeDim>, ElementId<VolumeDim>>,
-                          Package>& rhs) noexcept {
-        return mean_difference(lhs) < mean_difference(rhs);
-      });
+
+  double max_difference = -1.0;
```

Comment:
why not 0.0?  I'd prefer either 0.0 or the minimum allowable double

Reply:
I prefer the initial state to have an "invalid" value, which in this case means anything < 0. I'll replace with `std::numeric_limits<double>::lowest()`.